indico/modules/events/registration/templates/management/_registration_details.html

```diff
@@ -179,6 +180,13 @@
                     {% trans %}Get ticket{% endtrans %}
                 </a>
             {% endif %}
+            {% if can_get_document %}
+                <a href="#" class="i-button accept icon-agreement"
+                data-callback="printReceipts"
+                data-params="{{ {'registration_id': ['registration.id'], 'event_id': registration.registration_form.event.id} | tojson | forceescape }}">
```

Comment:
Did you test it? I'm quite sure you are passing a literal string `registration.id` instead of the actual ID...

Reply:
Done @ThiefMaster .
Sorry about that... 

The modal was opening perfectly, so I though it would work.